src/Tribe/Main.php

```diff
@@ -3826,76 +2990,6 @@ public function verify_unique_name( $name, $post_type ) {
 			return ( $results ) ? 0 : 1;
 		}
 
-		/**
-		 * Given a date (YYYY-MM-DD), returns the first of the next month
-		 * hat tip to Dan Bernadict for method cleanup
-		 *
-		 * @param string $date
-		 *
-		 * @return string Next month's date
-		 * @throws OverflowException
-		 */
-		public function nextMonth( $date ) {
-			if ( PHP_INT_SIZE <= 4 ) {
-				if ( date( 'Y-m-d', strtotime( $date ) ) > '2037-11-30' ) {
-					throw new OverflowException( esc_html__( 'Date out of range.', 'the-events-calendar' ) );
-				}
-			}
-
-			// Create a new date object: a badly formed date can trigger an exception - in such
-			// a scenario try again and default to the current time instead
-			try {
-			$date = new DateTime( $date );
-			}
-			catch ( Exception $e ) {
-				$date = new DateTime;
-			}
-
-			// set date object to be the first of the month -- all months have this day!
-			$date->setDate( $date->format( 'Y' ), $date->format( 'm' ), 1 );
-
-			// add a month
-			$date->modify( '+1 month' );
-
-			// return the year-month
-			return $date->format( 'Y-m' );
-		}
-
-		/**
-		 * Given a date (YYYY-MM-DD), return the first of the previous month
-		 * hat tip to Dan Bernadict for method cleanup
-		 *
-		 * @param string $date
-		 *
-		 * @return string Previous month's date
-		 * @throws OverflowException
-		 */
-		public function previousMonth( $date ) {
```

Comment:
Brought back and deprecated. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3178))